src/graphql/generateFlowTypes.js

```diff
@@ -48,12 +48,18 @@ export default function generateFlowTypes(
     }
   }
   function formatNodeField(field: NodeFieldType): string {
+    if (field.fidelity.type === "UNFAITHFUL") {
```

Comment:
Did you test the changes to `generateFlowTypes`?

I’m somewhat sympathetic to only lightly testing changes that consist of
adding error cases that are planned to be removed in a subsequent
commit, but these seem simple enough to test that you might as well do
it. Personally, I wouldn’t feel comfortable checking in functional
changes to a file that is already well-tested without updating the tests
at all.

Reply:
:+1: done